packages/styles/src/popover.scss

```diff
@@ -6,13 +6,31 @@
 $block: #{$fd-namespace}-popover;
 $blockBody: #{$fd-namespace}-popover__body;
 
+body.overlay-active::before {
+  content: '';
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100vw;
+  height: 100vh;
+  background-color: var(--fdModal_BackgroundColor);
+  z-index: 999;
+  pointer-events: all;
+}
+
 .#{$block} {
   @include fd-reset();
 
   position: relative;
   display: inline-block;
   max-width: 100%;
 
+  &__modal {
+    position: relative;
+    z-index: 1000;
+    pointer-events: auto;
```

Comment:
Why do you set explicitly the pointer-event to `auto`? This is the default value for the browsers:
<img width="1133" alt="Screenshot 2024-08-27 at 1 46 05 PM" src="https://github.com/user-attachments/assets/af33bba4-5740-4ea5-b360-9cbce224052b">

info: https://www.w3schools.com/cssref/css3_pr_pointer-events.php